static/app/utils/theme.tsx

```diff
@@ -692,39 +944,62 @@ const iconSizes = {
   xxl: `${iconNumberSizes.xxl}px`,
 } as const;
 
+// @TODO(jonasbadalic): This was missing profiles, profileChunks, profileDuration, spans, spansIndexed, uptime, what do we do with them?
+const dataCategory: Record<
+  Exclude<
+    DataCategory,
+    'profiles' | 'profileChunks' | 'profileDuration' | 'spans' | 'spansIndexed' | 'uptime'
+  >,
+  string
+> = {
+  [DATA_CATEGORY_INFO.error.plural]: CHART_PALETTE[4][3],
+  [DATA_CATEGORY_INFO.transaction.plural]: CHART_PALETTE[4][2],
+  [DATA_CATEGORY_INFO.attachment.plural]: CHART_PALETTE[4][1],
+  [DATA_CATEGORY_INFO.replay.plural]: CHART_PALETTE[4][4],
+  [DATA_CATEGORY_INFO.monitorSeat.plural]: '#a397f7',
+};
+
+/**
+ * Default colors for data usage outcomes
+ * @TODO(jonasbadalic): This was missing abuse and cardinality limited, what do we do with them?
+ */
+type OutcomeColors = Record<
+  Exclude<Outcome, Outcome.ABUSE | Outcome.CARDINALITY_LIMITED>,
+  string
+>;
+
+const outcome: OutcomeColors = {
+  [Outcome.ACCEPTED]: CHART_PALETTE[5][0], // #444674 - chart 100
+  [Outcome.FILTERED]: CHART_PALETTE[5][2], // #B85586 - chart 300
+  [Outcome.RATE_LIMITED]: CHART_PALETTE[5][3], // #E9626E - chart 400
+  [Outcome.INVALID]: CHART_PALETTE[5][4], // #F58C46 - chart 500
+  [Outcome.CLIENT_DISCARD]: CHART_PALETTE[5][5], // #F2B712 - chart 600
+  [Outcome.DROPPED]: CHART_PALETTE[5][3], // #F58C46 - chart 500
+};
+
+/**
+ * Values shared between light and dark theme
+ */
 const commonTheme = {
   isChonk: false,
-  breakpoints: {
-    xsmall: '500px',
-    small: '800px',
-    medium: '992px',
-    large: '1200px',
-    xlarge: '1440px',
-    xxlarge: '2560px',
-  },
+  breakpoints,
 
   ...lightColors,
-
   ...lightShadows,
 
+  // Icons
   iconSizes,
   iconNumberSizes,
-
-  iconDirections: {
-    up: '0',
-    right: '90',
-    down: '180',
-    left: '270',
-  },
+  iconDirections: iconDirectionToAngle,
```

Comment:
Seems like these should exist on icon component

Reply:
Agree, I just didnt want to change too much as I've already moved the entire file around :D